corim/testcases/src/corim-with-extensions.yaml

```diff
@@ -0,0 +1,75 @@
+# minimalist unsigned-corim that embeds comid.PSARefValJSONTemplate 
+# - profile (claim 3) has been set to "http://example.com/test-profile"
+# - extension claim -1 has been added to corim
+# - extension claim -1 has been added to comid entity
```

Comment:
It is not very clear how these extensions have been realised, can you please add a comment next to the extension. 

If I understand correctly, Line 3 comment applies to Line 8: where foo has been added?

However, for Entity, inside CoMID, -1 is a number, is it Reg-ID or something ?

May be work adding the YAML comment : Please..

Reply:
Well, the whole point is that they do not have pre-defined names, in the spec so -1 is just -1;

Reply:
(they do not have pre-defined numbers either, which is why -1, the first in the extension range, is used for both corim and comid entity.

Reply:
All I am saying is to state how you are using -1 in the code, as an example!. I am not sure it is very clear how extensions are exercised...

Reply:
This is not indened as an example of extension usage. This is a test case for a very specific test. The test just checks that the  code can access the extensions embedded in the struct heiarchy. It doesn't do anything with the extesions, and so there is no meaning attached to the extensions beyound "this is the extension with key -1 and value "foo"". 

 Bear in mind the the YAML files are intended purely as an easier means to getting at binary blobs for tests (and binary blobs is all we had before -- those are even less clear). We do not expect people using the extensions mechanim to be using YAML (they will be using the `cocli` "templates" or defining things programatically in code). There is a separate file (`corim/example_profile_test.go`) that actually contains example code intended to show how the extensions are defined and used.